src/pudl/extract/eia_bulk_elec.py

```diff
@@ -19,16 +19,6 @@
 
 from pudl.workspace.datastore import Datastore
 
-# Unfortunately, the date formats in the EIA bulk electricity data are not uniform,
-# and so for now we need to fall back on dateutil. Currently this warning is emitted
-# thousands of times, clogging up the logs. Just warn us once!
-warnings.filterwarnings(
-    action="once",
-    message="Could not infer format, so each element will be parsed individually, falling back to `dateutil`.",
-    category=UserWarning,
-    module="pudl.extract.eia_bulk_elec",
-)
-
```

Comment:
I could not for the life of me figure out how to get this to work. `action="once"` and `action="module"` still displayed all of the warnings, no matter where I put it in the module. So I ended up just ignoring the warnings entirely, and scoping that as narrowly as possible.